juju/testing/apiserver.go

```diff
@@ -555,6 +569,48 @@ func (s *ApiServerSuite) tearDownConn(c *gc.C) {
 	}
 }
 
+func (s *ApiServerSuite) SeedCAASCloud(c *gc.C) {
```

Comment:
Is this better off being in a CAAS test suite?

Reply:
So, maybe we should have a CAASApiServerSuite, which only offers these methods and doesn't pollute the ApiServerSuite? Then we can just seed the CAAS cloud automatically.

Reply:
@tlm I'm going to sweep back around to this in a new PR. Easier to review.

Reply:
Yeah that's fine. I have some ideas on it.